src/libAtomVM/bif.c

```diff
@@ -54,6 +55,11 @@
         RAISE_ERROR_BIF(fail_label, BADARG_ATOM);              \
     }
 
+#define SCHEDULER_ID_BITS 4
+#define MAX_SCHEDULERS 16
```

Comment:
We can't keep this. Several personal computers have more than 16 cores and generic unix or emscripten ports will hit this limit.

Reply:
It shouldn't be needed after we simplify the implementation.